recipes/argtable3/all/conanfile.py

```diff
@@ -100,18 +95,19 @@ def package_info(self):
         suffix = ""
         if not self.options.shared:
             suffix += "_static"
-        if tools.Version(self.version) >= "3.2.1" and self.settings.build_type == "Debug":
+        if Version(self.version) >= "3.2.1" and self.settings.build_type == "Debug":
             suffix += "d"
-        self.cpp_info.libs = ["argtable3{}".format(suffix)]
-        if not self.options.shared:
-            if self.settings.os in ("FreeBSD", "Linux"):
-                self.cpp_info.system_libs.append("m")
+        self.cpp_info.libs = [f"argtable3{suffix}"]
+        if self.settings.os in ("FreeBSD", "Linux"):
+            self.cpp_info.system_libs.append("m")
+
+        self.cpp_info.set_property("cmake_file_name", "Argtable3")
+        self.cpp_info.set_property("cmake_target_name", "argtable3")
```

Comment:
it's incorrect, there is a comment in this recipe explaining what are official target names:

> # These targets were for versions <= 3.2.0 (newer create argtable3::argtable3)